website/docs/workloads/specview/specview-profiles.md

```diff
@@ -0,0 +1,42 @@
+# SPECviewperf Workload Profiles
+The following profile runs the SPECviewperf Workloads.
+
+* [Workload Details](./specview.md)  
+
+## PERF-GPU-SPECVIEW-AMD.json
+Runs the stock SPECviewperf Workloads.
+
+* [Workload Profile](https://github.com/microsoft/VirtualClient/blob/main/src/VirtualClient/VirtualClient.Main/profiles/PERF-GPU-SPECVIEW-AMD.json) 
+
+* **Supported Platform/Architectures**
+  * win-x64
+  * AMD v620 GPU
+
+* **Supports Disconnected Scenarios**  
+  * No. Internet connection required.
+
+* **Dependencies**  
+  The dependencies defined in the 'Dependencies' section of the profile itself are required in order to run the workload operations effectively.
+  * Internet connection.
+
+  Additional information on components that exist within the 'Dependencies' section of the profile can be found in the following locations:
+  * [Installing Dependencies](https://microsoft.github.io/VirtualClient/docs/category/dependencies/)
+
+* **Profile Parameters**  
+NA
+
+* **Profile Runtimes**  
+  * The SPECviewperf package zip file is around 30GB. Downloading and extracting this file take about 30 minutes to complete. 
+  * The SPECviewperf workload takes about 30 minutes to run.
```

Comment:
Thank you for document the runtime. Is it all scenarios or each?

Reply:
Each viewset takes about 5 min to run on a machine with a single AMD v620 GPU. I will add this information in the doc to make it more clear